BridgeSDK/Internal/SBBCacheManager.m

```diff
@@ -675,19 +675,21 @@ - (NSManagedObjectContext *)cacheIOContext
 {
     if (!_cacheIOContext) {
         [self dispatchSyncToCacheManagerCoreDataQueue:^{
```

Comment:
Note: if we ever have a project where the design includes reseting the database frequently, then the queue should be moved to *always* being called to check for `nil` since there's a possible race condition in that code as well. Given our current login/withdrawal app design, this isn't really an issue.